src/components/ListItem.tsx

```diff
@@ -66,10 +74,11 @@ const Content = (props: ListItemProps) => {
 export const renderImageOrIcon = (
   size: number,
   image?: ImageRequireSource | string,
-  iconColor?: string
+  iconColor?: string,
+  style?: StyleProp<ViewStyle>
 ) => {
   if (typeof image === 'string') {
-    return <Icon name={image!} size={size} color={iconColor} />;
+    return <Icon name={image!} size={size} color={iconColor} style={style!} />;
   } else return <Image source={image!} style={styles.Action} />;
 };
 
```

Comment:
This modification is necessary, as we import `renderImageOrIcon` in `ChipLarge` to render also Images and not only Icons included in the .ttf file.

Reply:
This is not necessary any more, we modified ChipLarge to handle different type of logic, styles etc... and we duplicated this functionality in ChipLarge. The funcionlity differed toom much from the original one.

Reply:
a581316